<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDlg</class>
 <widget class="QDialog" name="ConfigDlg">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>157</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>157</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>157</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Linver configuration</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>124</y>
     <width>386</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>392</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="brandingV" stretch="1,1">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Branding style to use:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="brandingcmb">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxCount">
         <number>5</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Windows Vista</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Windows 7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Windows 8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Windows 8.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="fakeDetails">
      <property name="text">
       <string>Display fake version instead of OS version</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="displayFullName">
      <property name="text">
       <string>Display full name</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="brandingV_3" stretch="1,0,0">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>To customize texts, branding image and other options, click customize.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="customizeBtn">
        <property name="text">
         <string>Customize...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
